src/services/category.js

```
@@ -68,7 +73,10 @@ const categoryService = {
       throw error;
     }
   
-    return category;
+    return {
+      ...category,
+      totalOffers: 10
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)